src/Dropdown.js

```diff
@@ -18,14 +18,14 @@ class Dropdown extends Component {
   }
 
   render () {
-    const { children, trigger, ...props } = this.props;
+    const { children, trigger, className, ...props } = this.props;
     this.idx = 'dropdown_' + idgen();
     delete props.trigger;
 
     return (
       <span>
         { this.renderTrigger() }
-        <ul {...props} className='dropdown-content' id={this.idx}>
+        <ul {...props} className={`dropdown-content ${className}`} id={this.idx}>
```

Comment:
you can use the `cx` helper. e.g.
https://github.com/react-materialize/react-materialize/blob/master/src/CardPanel.js#L14